beanprice/price.py

```diff
@@ -762,7 +776,7 @@ def process_args():
     if args.expressions:
         # Interpret the arguments as price sources.
         for source_str in args.sources:
-            psources = []
+            psources: List[PriceSource] = []
```

Comment:
Same comment as for the other instances.

Reply:
If removed, `mypy` fails with:

```
beanprice/price.py:782: error: Need type annotation for 'psources' (hint: "psources: List[<type>] = ...")
```